### PR TITLE
add global hub api

### PR DESCRIPTION
**Pull request by yanmxa**
Signed-off-by: myan [myan@redhat.com](mailto:myan@redhat.com)
Reference: https://issues.redhat.com/browse/ACM-9275

### CODE REVIEW

File: apis/multiclusterglobalhub.json.adoc [hunk 1]
```diff
@@ -0,0 +1,415 @@
+[#multiclusterglobalhub-api]
+= Multiclusterglobalhub API
+
+
+[[_rhacm-docs_apis_multiclusterglobalhub_jsonoverview]]
+== Overview
+This documentation is for the Multiclusterglobalhub resource for {product-title}. The Multiclusterglobalhub resource has four possible requests: create, query, delete and update.
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]: What do you think about this rewrite to make it more user focused?

```suggestion
Use the `Multiclusterglobalhub` resource for {product-title}. The `Multiclusterglobalhub` resource has four possible requests: create, query, delete, and update. 
```

File: apis/multiclusterglobalhub.json.adoc [hunk 2]
```diff
@@ -0,0 +1,415 @@
...
+
+=== URI scheme
+[%hardbreaks]
+__BasePath__ : /kubernetes/apis
+__Schemes__ : HTTPS
+
+
+=== Tags
+
+* multiclusterglobalhubs.operator.open-cluster-management.io/v1alpha4: Create and manage multiclusterglobalhubs
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]: Am I correct that this `Tag` could be written like the following?: 

```suggestion
* `multiclusterglobalhubs.operator.open-cluster-management.io/v1alpha4`: Create and manage `multiclusterglobalhubs`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 3]
```diff
@@ -0,0 +1,415 @@
...
+=== Create a multiclusterglobalhub
+....
+
+POST /operator.open-cluster-management.io/v1alpha4/namespaces/{namespace}/multiclusterglobalhubs/{multiclusterglobalhub_name}
+
+....
+
+
+==== Description
+Create a multiclusterglobalhub.
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* Create a `multiclusterglobalhub`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 4]
```diff
@@ -0,0 +1,415 @@
...
+|*403*|Access forbidden|No Content
+|*404*|Resource not found|No Content
+|*500*|Internal service error|No Content
+|*503*|Service unavailable|No Content
+|===
+
+
+==== Consumes
+
+* `application/json`
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]: To be consistent with the other bullet lists, maybe we should start with an action verb for users to follow? What do you think of the following suggestion?: 

```suggestion
* Consume your `multiclusterglobalhub` with the following: `application/json`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 5]
```diff
@@ -0,0 +1,415 @@
...
+
+
+==== Tags
+
+* operator.open-cluster-management.io
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]: Same comment as above. It seemed at the beginning of this doc that we wanted to give the user some action to follow with the Tags? What do you think of this suggestion?: 

```suggestion
* Give your `multiclusterglobalhub` a tag with the following: `operator.open-cluster-management.io`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 6]
```diff
@@ -0,0 +1,415 @@
...
+=== Query all multiclusterglobalhubs
+....
+
+GET /operator.open-cluster-management.io/v1alpha4/namespaces/{namespace}/multiclusterglobalhubs
+
+....
+
+
+==== Description
+Query your multiclusterglobalhubs for more details.
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* Query your `multiclusterglobalhubs` for more details.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 7]
```diff
@@ -0,0 +1,415 @@
...
+|*403*|Access forbidden|No Content
+|*404*|Resource not found|No Content
+|*500*|Internal service error|No Content
+|*503*|Service unavailable|No Content
+|===
+
+
+==== Consumes
+
+* `application/json`
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* `Consume your `multiclusterglobalhub` with `application/json`. 
```

File: apis/multiclusterglobalhub.json.adoc [hunk 8]
```diff
@@ -0,0 +1,415 @@
...
+
+
+==== Tags
+
+* operator.open-cluster-management.io
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* Give your `multiclusterglobalhub` a tag with the following: `operator.open-cluster-management.io`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 9]
```diff
@@ -0,0 +1,415 @@
...
+=== Query a single multiclusterglobalhub
+....
+
+GET /operator.open-cluster-management.io/v1alpha4/namespaces/{namespace}/multiclusterglobalhubs/{multiclusterglobalhub_name}
+
+....
+
+
+==== Description
+Query a single multiclusterglobalhub for more details.
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
Query a single `multiclusterglobalhub` for more details.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 10]
```diff
@@ -0,0 +1,415 @@
...
+|*403*|Access forbidden|No Content
+|*404*|Resource not found|No Content
+|*500*|Internal service error|No Content
+|*503*|Service unavailable|No Content
+|===
+
+
+==== Tags
+
+* operator.open-cluster-management.io
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* Give your `multiclusterglobalhub` a tag with the following: `operator.open-cluster-management.io`.
```

File: apis/multiclusterglobalhub.json.adoc [hunk 11]
```diff
@@ -0,0 +1,415 @@
...
```
Comment (jc-berger) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
* Give your `multiclusterglobalhub` a tag with the following: `operator.open-cluster-management.io`.
```

File: apis/multiclusterglobalhub.json.adoc (hunk 1 above)
Comment (yanmxa) [reply apis/multiclusterglobalhub.json.adoc]: "to improve your cluster development" It seems a little wired, Is this your intent?

File: apis/multiclusterglobalhub.json.adoc (hunk 2 above)
Comment (yanmxa) [reply apis/multiclusterglobalhub.json.adoc]: I'm not sure. @ldpliu @clyang82 What do you think of it?

Comment (ldpliu) [reply apis/multiclusterglobalhub.json.adoc]: I propose set it with same as other format of apis.
like: https://github.com/stolostron/rhacm-docs/blob/db04d247dfa4173160ddc9d444a6313e99625463/apis/clusterset.json.adoc#tags
So I suggest use `* multiclusterglobalhubs.operator.open-cluster-management.io/v1alpha4: Create and manage multiclusterglobalhubs`

File: apis/multiclusterglobalhub.json.adoc (hunk 1 above)
Comment (jc-berger) [reply apis/multiclusterglobalhub.json.adoc]: @yanmxa thanks for responding! I removed it. 

Basically, my intent was the "why" for the user. Like, why they'd use `MultiClusterHub` or what is the benefit. If we can't think of a simple way to say that, I think we can keep it pretty minimal and concise, like the new rewrite I did. 

Let me know what you think, thanks!

File: apis/multiclusterglobalhub.json.adoc (hunk 2 above)
Comment (jc-berger) [reply apis/multiclusterglobalhub.json.adoc]: @yanmxa and @ldpliu thanks! I made the change as you requested :)

**Comment by clyang82** [comment]
/hold
see the discussion in https://issues.redhat.com/browse/ACM-9275

**Review by jc-berger — APPROVED** [review]
/lgtm

File: apis/multiclusterglobalhub.json.adoc [hunk 12]
```diff
@@ -0,0 +1,415 @@
...
+
+
+[[_rhacm-docs_apis_multiclusterglobalhub_jsonoverview]]
+== Overview
+Use the `Multiclusterglobalhub` resource for {product-title}. The `Multiclusterglobalhub` resource has four possible requests: create, query, delete, and update. 
+
+
+=== Version information
+[%hardbreaks]
+__Version__ : 2.9.0
```
Comment (dockerymick) [inline apis/multiclusterglobalhub.json.adoc]:
```suggestion
__Version__ : 2.10.0
```

**Pull request closed without merging**